fbpic/fields/fields.py

```diff
@@ -216,8 +216,8 @@ def __init__( self, Nz, zmax, Nr, rmax, Nm, dt, zmin=0.,
             #Create the envelope interpolation grids for each modes
             #The envelope modes range from -Nm + 1 to Nm - 1
             self.envelope_interp = []
-            self.envelope_mode_numbers = [ m for m in range(self.Nm) ] + \
-                                         [ m for m in range(-self.Nm+1, 0)]
+            self.envelope_mode_numbers = np.array(
+              [m for m in range(self.Nm)] + [m for m in range(-self.Nm+1,0)])
```

Comment:
Converting `envelope_mode_numbers` to an array is important, because it is then passed to some GPU kernels later. The numba GPU kernels support arguments that are arrays, but not lists.